### PR TITLE
View image detail by click and 'enter'

### PR DESCRIPTION
**Pull request by yipenghe**
#1934 
Open/click bundle detail when clicked on image item or hit 'enter' when focused

### CODE REVIEW

File: frontend/src/components/worksheets/items/ImageItem.js [hunk 1]
```diff
@@ -1,23 +1,49 @@
...
+            showDetail: false,
+            bundleInfoUpdates: {},
+        };
     }
 
+    receiveBundleInfoUpdates = (update) => {
+        let { bundleInfoUpdates } = this.state;
+        // Use object spread to update.
+        bundleInfoUpdates = { ...bundleInfoUpdates, ...update };
+        this.setState({ bundleInfoUpdates });
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/ImageItem.js]: I think this would be clearer
```suggestion
        this.setState({ bundleInfoUpdates: { ...bundleInfoUpdates, ...update } });
```

File: frontend/src/components/worksheets/items/ImageItem.js [hunk 2]
```diff
@@ -1,23 +1,49 @@
...
+                (e) => {
+                    e.preventDefault();
+                    if (!this.props.confirmBundleRowAction(e.code)) {
+                        this.setState({ showDetail: !this.state.showDetail });
+                    }
+                },
+                'keydown',
+            );
+            // unbind shortcuts that are active for markdown_block and worksheet_block
+            Mousetrap.unbind('i');
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/ImageItem.js]: Is `i` the only shortcut that needs to be unbound here?

File: frontend/src/components/worksheets/items/ImageItem.js [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+        }
+        const { classes } = this.props;
+        var item = this.props.item;
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/ImageItem.js]:
```suggestion
        const item = this.props.item;
```

File: frontend/src/components/worksheets/items/ImageItem.js [hunk 4]
```diff
@@ -1,23 +1,49 @@
...
+        var bundleInfo = item.bundles_spec.bundle_infos[0];
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/ImageItem.js]:
```suggestion
        const bundleInfo = item.bundles_spec.bundle_infos[0];
```

File: frontend/src/components/worksheets/items/ImageItem.js [hunk 5]
```diff
@@ -27,12 +53,28 @@ class ImageItem extends React.Component {
         if (this.props.item.hasOwnProperty('width')) {
             styles['width'] = this.props.item.width + 'px';
         }
-
         return (
-            <div className='ws-item' onClick={this.handleClick}>
-                <div className={className} ref={this.props.item.ref}>
+            <div className='ws-item'>
+                <div className={className} ref={this.props.item.ref} onClick={this.handleClick}>
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/ImageItem.js:56]: Why did you move the `onClick` handler to the child?

Comment (yipenghe) [reply frontend/src/components/worksheets/items/ImageItem.js:56]: Otherwise the click will trigger when we are clicking on the bundle detail

File: frontend/src/components/worksheets/items/ImageItem.js (hunk 2 above)
Comment (yipenghe) [reply frontend/src/components/worksheets/items/ImageItem.js]: Actually it's not needed...copied from the bundle row implementation
